src/manager/InMemoryTaskManager.java

```diff
@@ -151,22 +214,14 @@ public Epic findEpicById(Integer id) {
 
     private void updateEpicStatus(Epic epic) {
         List<Subtask> subtasksOfEpic = getSubtasksOfEpic(epic.getId());
-        boolean allNew = true;
-        boolean allDone = true;
 
         if (subtasksOfEpic.isEmpty()) {
             epic.setStatus(Status.NEW);
             return;
         }
 
-        for (Subtask subtask : subtasksOfEpic) {
-            if (subtask.getStatus() != Status.NEW) {
-                allNew = false;
-            }
-            if (subtask.getStatus() != Status.DONE) {
-                allDone = false;
-            }
-        }
+        boolean allNew = subtasksOfEpic.stream().allMatch(s -> s.getStatus() == Status.NEW);
```

Comment:
Стримы принято бить по строкам, чтобы сделать код более читаемым. Также в стримах лучше не использвать однобуквенные названия переменных, так как они делают код менее понятным

```suggestion
        boolean allNew = subtasksOfEpic.stream()
            .allMatch(subtask -> subtask.getStatus() == Status.NEW);
```

Поправь следующим образом все стримы в коде